site/content/install/_index.md

```diff
@@ -13,7 +13,9 @@ Installing and running `getenvoy` on Linux and macOS is as easy as:
 
 ```sh
 $ curl -L https://getenvoy.io/install.sh | bash -s -- -b /usr/local/bin
-$ getenvoy run --version
```

Comment:
I thought doing a version command was a punt and doesn't help anyone so changed it